exercises/atbash-cipher/.meta/solutions/atbash_cipher.rb

```diff
@@ -1,16 +1,24 @@
-class Atbash
+class AtbashCipher
   def self.encode(plaintext)
     new(plaintext).encode
   end
 
-  attr_reader :plaintext
+  def self.decode(ciphertext)
+    new(ciphertext).decode
+  end
+
+  attr_reader :text
 
-  def initialize(plaintext)
-    @plaintext = plaintext
+  def initialize(text)
+    @text = text
   end
 
   def encode
-    chunk convert(normalize(plaintext))
+    chunk convert(normalize(text))
+  end
+
+  def decode
+    convert(normalize(text))
```

Comment:
It smells like `convert(normalize(text))` would be better off stored in a method, or `convert` could be `normalize(text).tr(alphabet, key)`, so that `encode` could be `chunk convert` and `decode` could be `convert`.

Thoughts?

I just didn't want to make drastic changes unless you all thought they were necessary :smile: